### PR TITLE
Init package content, initial tests passing

### PR DESCRIPTION
**Pull request by bluegeo**
Added initial package content, including tests. Tests are passing, although collection creation is still WIP.

### CODE REVIEW

**Comment by cuttlefish** [comment]
Just noticed this as I was about to hit merge on #1. That would have been ugly. Let's stick with this one.
I was going to push some formatting fixes so the actions can run, but I don't seem to have permissions on the sparkgeo fork.

File: README.md [hunk 1]
```diff
@@ -1,17 +1,42 @@
...
-4. Rewrite this README to provide information about how to use your package.
-5. Update the LICENSE with your company's information (or whomever holds the copyright).
-6. Run `sphinx-quickstart` in the `docs` directory to create the documentation template.
+# stactools aafc-landuse
+
+## Agriculture and Agri-Food Canada Land Use
+
+### Description
+
+The Land Use (LU) maps cover all areas of Canada south of 600N at a spatial resolution of 30 metres. The LU classes follow the protocol of the Intergovernmental Panel on Climate Change (IPCC) and consist of: Forest, Water, Cropland, Grassland, Settlement and Otherland.
```
Comment (jbants) [inline README.md]: `The Land Use (LU) maps cover all areas of Canada south of 600N` Should this be 60degN?

File: src/stactools/aafc_landuse/__init__.py [hunk 2]
```diff
@@ -0,0 +1,12 @@
+import stactools.core
+
```
Comment (jbants) [inline src/stactools/aafc_landuse/__init__.py:6]: Add `__all__ `

File: src/stactools/aafc_landuse/constants.py [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+LANDUSE_CRS = CRS.from_epsg(LANDUSE_EPSG)
+LANDUSE_TITLE = "Land Use 1990, 2000 & 2010"
+LICENSE = "OGL-Canada-2.0"
+LICENSE_LINK = Link(
+    rel="license",
+    target="https://open.canada.ca/en/open-government-licence-canada",
+    title="Open Government Licence - Canada",
+)
+
+DESCRIPTION = """The 1990, 2000 and 2010 Land Use (LU) maps cover all areas of Canada south of 600N at a spatial resolution of 30 metres. The LU classes follow the protocol of the Intergovernmental Panel on Climate Change (IPCC) and consist of: Forest, Water, Cropland, Grassland, Settlement and Otherland."""
```
Comment (jbants) [inline src/stactools/aafc_landuse/constants.py]: 60degN

File: README.md (hunk 1 above)
Comment (bluegeo) [reply README.md]: That was copied from here https://open.canada.ca/data/en/dataset/9e1efe92-e5a3-4f70-b313-68fb1283eadf, but yes...it should be

**Review by jbants — APPROVED** [review]

**Pull request merged**